examples/buildctl-daemonless/buildctl-daemonless.sh

```diff
@@ -19,7 +19,7 @@ set -eu
 # * addr
 # * log
 tmp=$(mktemp -d /tmp/buildctl-daemonless.XXXXXX)
-trap "kill \$(cat $tmp/pid); rm -rf $tmp" EXIT
+trap "kill \$(cat $tmp/pid); wait \$(cat $tmp/pid) || true; rm -rf $tmp" EXIT
```

Comment:
Can we have `timeout`?

Reply:
Do you have an example in mind?

Reply:
I guess with timeout we should timeout until we send sigkill instead?